.github/workflows/bindings_python.yml

```diff
@@ -40,6 +40,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+        with:
+          working-directory: "bindings/python"
+          run: |
```

Comment:
Hi, this section seems to added into `actions/checkout@v4` wrongly.

We need to add a new step here like:

```yaml
- name: Ruff Check
  working-directory: "bindings/python"
  run: |
    pip install ruff
    ruff check .
```